### ISSUE TITLE
Should there be feedback for no exact match

### ISSUE THREAD

**Issue by jimentwood** [cul-it/arxiv-search#82]
Ran into this by mistake.

If you do a strict search "reimann hypothesis" that contains a typo the search is smart enough to return related results. But there is no warning that there were no exact matches.

I don't know that this actually needs to be changed so this is more of a question... 

Can we imagine use cases of searching for a specific term or phrase and wanting to see only exact matches?

Should there be a warning that the exact search was not found and so we are displaying related results?

https://beta.arxiv.org/search/advanced?terms-0-operator=AND&terms-0-term=%22reimann+hypothesis%22&terms-0-field=title&terms-1-operator=AND&terms-1-term=proof&terms-1-field=title&classification-physics_archives=all&date-year=&date-from_date=&date-to_date=&size=25

**Comment by eawoods** [comment]
Yes, we should definitely return a message about no exact results - especially since the engine is doing a related-results behavior. It's best practice and has some grounds in (human) search behavior research.

**Comment by erickpeirson** [comment]
This behavior may have changed with recent work. Let's see what it looks
like with a fresh deploy
